cmd/bosun/web/static/js/ace/mode-bosun.js

```diff
@@ -0,0 +1,167 @@
+ace.define("ace/mode/bosun_highlight_rules",["require","exports","module","ace/lib/oop","ace/mode/text_highlight_rules"], function(require, exports, module) {
```

Comment:
Do any of these requires depend on files we didn't include? I'm not seeing 404s but it looks like maybe something is missing?